optimize/self-managed/optimize-deployment/configuration/history-cleanup.md

```diff
@@ -101,7 +95,7 @@ historyCleanup:
 
 <TabItem value='ingestedevent'>
```

Comment:
Can we delete this tab item please. This is event based process related, which is not supported in C8 anyway